moonraker/components/proc_stats.py

```diff
@@ -304,11 +307,9 @@ def _get_cpu_thermal_file(self) -> str:
                 hwmon_name = pathlib.Path(hwmon.path + "/name")
                 try:
                     name = hwmon_name.read_text().strip()
-                    # k10temp     : AMD CPU
-                    # coretemp    : intel cpu
-                    # cpu_thermal : raspberry pi
-                    if name == 'k10temp' or name == 'coretemp' or name == \
-                            'cpu_thermal':
+                    if name == HWMON_AMD_NAME \
+                            or name == HWMON_INTEL_NAME \
+                            or name == HWMON_RPI_NAME:
```

Comment:
We can simplify this condition and make it easily extensible with something like the following.  First replace the new constants with a `dict` mapping the platform names:

```python
HWMON_PLATFORMS = {
    "coretemp": "Intel",
    "k10temp": "AMD",
    "cpu_thermal": "Raspberry Pi" 
}

```

Then change the condition.  As a bonus we can log the detected platform.

```python
if name in HWMON_PLATFORMS:
    pf = HWMON_PLATFORMS[name]
    logging.info(f"Monitoring temperature for {pf} CPU")
    return hwmon.path + "/temp1_input"
```